var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -370,6 +370,14 @@ def _microarch_target_args(self):
             # case can go away.
             args.append("TARGET=" + "RISCV64_GENERIC")
 
+        elif self.spec.satisfies("@0.3.19: target=a64fx"):
+            # Special case for Fujitsu's A64FX
+            if any(self.spec.satisfies(i) for i in ["%gcc@11:", "%clang", "%fj"]):
+                args.append("TARGET=A64FX")
+            else:
+                # fallback to armv8-a+sve without -mtune=a64fx flag
+                args.append("TARGET=ARMV8SVE")
```

Comment:
We can focus on a64fx in this PR, but as a follow up PR we can use this target also `if "sve" in microarch.features`, since now there are some other chips != a64fx which have SVE.